.github/workflows/build-images-base.yaml

```diff
@@ -207,4 +207,4 @@ jobs:
           username: ${{ secrets.IMG_REGISTRY_USERNAME }}
           password: ${{ secrets.IMG_REGISTRY_TOKEN }}
       - name: Print Image URL
-        run: echo "Image pushed to ${{ steps.push-to-quay.outputs.registry-paths }}"
+        run: echo "Image pushed to ${{ steps.push-to-quay.outputs.registry-paths }}"
```

Comment:
nitpick: lets keep this line break so that the PR doesn't touch this file since it's doesn't change anything other than removing a line break